### ISSUE TITLE
upload the sTeX plugin to CPAN

### ISSUE THREAD

**Issue by kohlhase** [KWARC/LaTeXML-Plugin-sTeX#73]
@dginev said 
> To register the plugin centrally with CPAN, it indeed needs to be submitted, which I can either do for you, or you can obtain a PAUSEID and do by hand (the UI there is a bit arcane): http://pause.perl.org

I think that we should make regular releases to CPAN, now that we have (some) tests. I am assigning this to Hang and hope that @dginev will supervise this.

**Comment by kohlhase** [comment]
The CPAN installation should be documented in ```README.md``` (see #74)

**Comment by angerhang** [comment]
I just requested to register as a developer, but am wondering if we already have a developer account that manages the submission which we should use instead?

**Comment by kohlhase** [comment]
I am sure Deyan has one, but you should have one as well.

**Comment by dginev** [comment]
Btw CPAN makes it somewhat hard and clunky to transfer ownership, and once you have the namespace it is considered yours, so it may be wise to have the longest-term maintainer upload the plugin. I've never transfered ownership so far, but know it's possible.

**Comment by kohlhase** [comment]
hmmm, then I guess that I should do that (or at least make an account in the KWARC name?)

**Comment by angerhang** [comment]
Yeah that sounds better. Having a KWARC name CPAN account reduces future submission problems. After the registration being accepted, Michael can just pass the credential to whoever is doing the actual uploading (I guess it would be me for this time).

**Comment by kohlhase** [comment]
so can you register KWARC?

**Comment by angerhang** [comment]
Since 

> Registered developers have a unique username and a home directory in the authors/id/ tree of CPAN. The write access to that directory is password protected.

I probably will just put `Michael Kohlhase` as author name (they require a proper civic name) and `KWARC` as id then. This account can be used as the primary maintainer and if someone else needs to upload in the future, there is an option for the primary maintainer to add co-maintainers.

**Comment by kohlhase** [comment]
good, then add yourself as a co-maintainer and work with your own account.

**Comment by angerhang** [comment]
I think we are almost ready to do a release in the `cpan` branch. Just need to finish the `readme.md`. We shall use the account that I created for Miko to do the first submission so that this account gets to be the primary developer so I will add other people to be the co-maintainers after the submission is accepted.

One thing that doesn't make sense to me is that the current version number for `LaTeXML-Plugin-sTeX` we have the `0.2` instead of `0.1` see https://github.com/KWARC/LaTeXML-Plugin-sTeX/blob/master/Makefile.PL#L16. But I can't find any previous release on cpan for this plugin. 

Wouldn't it make more sense to use `0.1` for the first release instead?

**Comment by kohlhase** [comment]
I do not care about the version number too much. I would probably go for 0.9 and hope to get all the current issues ironed out and call it 1.0 then. This is based on sTeX 1.4.
